### ISSUE TITLE
Merge JSON2CSV with JSONComparison script

### ISSUE THREAD

**Issue by karenmajewicz** [BTAA-Geospatial-Data-Project/dcat-metadata#11]
If there is no comparison script, harvest all instead of discontinuing

**Comment by smit1975** [comment]
Hi.   @karenmajewicz @mkernik @ruetz007 

I uploaded a Python 3 script to the 'dcat-metadata' master repository.  I saved both the Jupyter Notebook (*.ipynb) and standard Python (*.py) formats.  File is called "JsonHarvest_CodeCompilation_4".

I wouldn't use it to do any work - it's merely for review at this time.  I've been chipping away at for a few dozen hours in the last month and have a lot of content - some is refined and some is 'heinous' - and some is just wild random code snippets that looked cool - in comments.

Will be continuing to craft it until a mostly functioning sneakily robust script is ready for official roll-out by Feb. 1st.  This is 'soft', to 'quite soft', roll-out...

-Andrew
